src/reactapp/src/components/common/ToggleBox/ToggleBox.jsx

```diff
@@ -3,19 +3,20 @@ import { bool, node } from 'prop-types';
 
 import Header from '../Header';
 import { ArrowUpIcon, ArrowDownIcon } from '../icons';
+import {ArrowSmDownIcon, ArrowSmUpIcon} from "@heroicons/react/outline";
```

Comment:
Why we are preferring `@heroicons` here? The icons locally included in the repo are taken from the hero icons site itself. The idea behind creating icon components in the app was to avoid package inclusion.

Also, if `@heroicons` going to be used, then the line 5 import is unnecessary here.